<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>382</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>611</width>
    <height>382</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>611</width>
    <height>382</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="textEdit_words">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>291</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_text">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>291</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Текст</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Слова</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_solve">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Выполнить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_clear">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>270</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_task_1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>334</y>
     <width>541</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>В заданном русском тексте выбрать слова, которые без искажения могут быть написаны</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_task_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>357</y>
     <width>421</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>латинскимии буквами, например: СВЕТА РОЕТ РОВ, ВОВКА СЕЕТ ОВЁС</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>323</y>
     <width>611</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
